tasking/pulp/tasking/base.py

```diff
@@ -350,7 +350,7 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         :param einfo:   celery's ExceptionInfo instance, containing serialized traceback.
         :type einfo:    ???
         """
-        if isinstance(exc, PulpCodedException):
+        if isinstance(exc, PulpException):
```

Comment:
I agree that celery should be logging the exception before we even get into this method.